### PR TITLE
add: rearrange flexcontainer, img-component

### LINKED ISSUE #1: US 1: create travel-detail page
## Value proposition

As a user
I want to see an overview of my trip. 
So that I can see all important conditions at once.

## Description

image

## Acceptance criteria

- You can see a heading thats called " travel details "   
- An image of the destination ( city )
- A container with the following infos is displayed
- The type of travel / travel-reason
- Name of the city with a red pin icon on the left
- Underneath you can see a Box with two travel information-details displayed 
- The left box contains a person-icon and the names of the members, who join the respective trip
- The right box shows a heading called "time-period"
- The right box displays a time-period 

## Tasks

- [x] create 2 arrays
    - [x] one for the user.data / one for the trip.data
    - [x] both arrays contain two objects
- [x] create a styled.component 
- [x] create a flex-container
    - [x] align the following items "centered" in a column  
    - [x] h1 heading "travel details"
    - [x] create an image-element  
    - [x] add a image of berlin to it
    - [x] create a div-element that contains: 
    - [x] nest a h3-element in it with the value of "travel-reason"
    - [x] nest a p-element with the value of "travel destionation" with a pin-emoji  
- [x] create a grid-container that contains:
    - [x] use styled-components within the js.file to target the container with grid
    - [x] the fr should be arranged as displayed
    - [x] the name.value of "User1"
    - [x] add the icon you see on the wireframe
    - [x] add a h3-element with the text "time-period"
    - [x] import the travel-time-period from the array

## Size
Large

### PR DESCRIPTION
**Pull request by KevinKyaterekera**
US1: turned the grid-container into a flex-container and aligned all components in a centered column. build the TravelDetails-page in index.js. All traveldetails are took out of small array wich contains generell information about the upcoming trip. Username/-count will be applied later in this project.

### CODE REVIEW

**Comment by milodevhh** [comment]
Hey Kevin, make sure to use git switch -c the other commits went on main. Please see my comments in the Code.

File: components/TravelDetails/index.js [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
-      })}
-    </article>
+    <FlexContainer>
+      <article>
+        {trip.name}
+        <br />
+        📍{trip.destination}
+      </article>
+      <FlexArticle>
+        <UserIcon />
```
Comment (milodevhh) [inline components/TravelDetails/index.js:17]: In your US you mentioned - "The left box contains a person-icon and the names of the members, who join the respective trip" - please add the name of the Person.

File: components/DestinationImage/index.js [hunk 2]
```diff
@@ -3,17 +3,15 @@ import { styled } from "styled-components";
```
Comment (milodevhh) [inline components/DestinationImage/index.js:3]: If you want to work with images, better you put the in a lib or public Folder.

**Review by milodevhh — COMMENTED** [review]
Cool Start, I like the first impression via vercel - Keep on Going

**Review by florianblaschke — APPROVED** [review]
LGTM

**Pull request merged**